configs/efficientdet.py

```diff
@@ -0,0 +1,146 @@
+# model settings
+input_size = 512
+model = dict(
+    type='ATSS',
+    backbone=dict(
+        type='efficientnet_b0b',
+        out_indices=(3, 4, 5),
+        frozen_stages=-1,
+        norm_eval=False,
+        pretrained=True,
+        verbose=False,
+    ),
+    neck=dict(
+        type='BiFPN',
+        in_channels=[40, 112, 320],
+        out_channels=128,
+        num_outs=5,
+        num_layers=3,
+        # start_level=0,
+        # add_extra_convs=True,
+        # extra_convs_on_inputs=False,  # use P5
+        # num_outs=5,
+        # relu_before_extra_convs=True
+    ),
+    bbox_head=dict(
+        type='ATSSHead',
+        num_classes=81,
+        in_channels=128,
+        stacked_convs=3,
+        feat_channels=128,
+        octave_base_scale=8,
+        scales_per_octave=1,
+        anchor_ratios=[1.0],
+        anchor_strides=[8, 16, 32, 64, 128],
+        target_means=[.0, .0, .0, .0],
+        target_stds=[0.1, 0.1, 0.2, 0.2],
+        loss_cls=dict(
+            type='FocalLoss',
+            use_sigmoid=True,
+            gamma=2.0,
+            alpha=0.25,
+            loss_weight=1.0),
+        loss_bbox=dict(type='GIoULoss', loss_weight=2.0),
+        loss_centerness=dict(
+            type='CrossEntropyLoss', use_sigmoid=True, loss_weight=1.0)))
+# training and testing settings
+cudnn_benchmark = True
+train_cfg = dict(
+    assigner=dict(type='ATSSAssigner', topk=9),
+    allowed_border=-1,
+    pos_weight=-1,
+    debug=False)
+test_cfg = dict(
+    nms=dict(type='nms', iou_thr=0.5),
+    min_bbox_size=0,
+    score_thr=0.02,
+    max_per_img=200)
+# model training and testing settings
+# dataset settings
+dataset_type = 'CocoDataset'
+data_root = '/home/ikrylov/datasets/coco/'
+img_norm_cfg = dict(mean=[0, 0, 0], std=[255, 255, 255], to_rgb=True)
```

Comment:
Pytorchcv backbones are usually trained with other means and stds. Are you sure that it does not affect models' quality?